services/base.spec.js

```diff
@@ -145,14 +146,14 @@ describe('BaseService', () => {
 
     it('handles the request', async () => {
       expect(mockHandleRequest).to.have.been.calledOnce;
-      const requestHandler = mockHandleRequest.getCall(0).args[0];
+      const { handler: requestHandler } = mockHandleRequest.getCall(0).args[0];
 
       const mockSendBadge = sinon.spy();
       const mockRequest = {
         asPromise: sinon.spy(),
       };
       await requestHandler(
-        /*data*/ {},
+        /*queryParams*/ { suffix: '?' },
```

Comment:
What do you think about extracting the first and the second arguments to variables?
```js
      const queryParams = { suffix: '?' };
      const match = '/foo/bar.svg'.match(expectedRouteRegex);
      await requestHandler(
        queryParams,
        match,
        mockSendBadge,
        mockRequest
      );
```